source/developers/file-transfer-protocol.rst

```diff
@@ -0,0 +1,228 @@
+
+The FTP protocol does not require any of the ``admin:*`` `permissions <minio-policy-mc-admin-actions>`.
+The FTP protocols do not support any of the MinIO admin actions.
+
+Other MinIO Features
+~~~~~~~~~~~~~~~~~~~~
+
+The FTP protocol presents an additional means of accessing objects.
+The underlying MinIO service continues to work as expected otherwise.
+
+Other MinIO features, such as :ref:`server-side bucket replication <minio-bucket-replication-serverside>`, :ref:`site replication <minio-site-replication-overview>`, or :ref:`server-side encryption <minio-sse>`, continue to work as is.
+
+Prerequisites
+-------------
+
+- MinIO RELEASE.2023-04-20T17-56-55Z or later.
+- Enable an FTP or SFTP port for the server.
+
+Procedure
+---------
+
+1. Start MinIO with an FTP and/or SFTP port enabled.
+
+   .. code-block:: shell
+      :class: copyable
+
+      minio server http://server{1...4}/disk{1...4}                               \
+      --ftp="address=:8021" --ftp="passive-port-range=30000-40000"                \
+      --sftp="address=:8022" --sftp="ssh-private-key=/home/miniouser/.ssh/id_rsa" \
+      ...
+    
+   See the :mc-cmd:`minio server --ftp` and :mc-cmd:`minio server --sftp` for details on using these flags to start the MinIO service.
+
+   The output of the command should returns a response that includes something like the following:
+
+   .. code-block:: shell
+
+      MinIO FTP Server listening on :8021
+      MinIO SFTP Server listening on :8022
+
+2. Use your preferred ftp client to connect to the MinIO deployment with the user whose policies you want to use to access buckets and objects.
+
+   The specifics of connecting to the MinIO deployment depend on your FTP client.
+   Refer to the documentation for your client.
+
+   To connect over TLS or through SSH, you must use a client that supports the desired protocol.
+
+Examples
+--------
+
+The examples here use the ``ftp`` CLI client on a Linux system.
+
+Connect to an FTP Server
+~~~~~~~~~~~~~~~~~~~~~~~~
+
+The following example Connect to a server using ``minio`` credentials to list contents in a bucket named ``runner``
+
+.. code-block:: shell
+
+   > ftp localhost -P 8021
+   Connected to localhost.
+   220 Welcome to MinIO FTP Server
+   Name (localhost:user): minioadmin
+   331 User name ok, password required
+   Password:
+   230 Password ok, continue
+   Remote system type is UNIX.
+   Using binary mode to transfer files.
+   ftp> ls runner/
+   229 Entering Extended Passive Mode (|||39155|)
+   150 Opening ASCII mode data connection for file list
+   drwxrwxrwx 1 nobody nobody            0 Jan  1 00:00 chunkdocs/
+   drwxrwxrwx 1 nobody nobody            0 Jan  1 00:00 testdir/
+   ...
+
+Download an Object
+~~~~~~~~~~~~~~~~~~
+
+The example continues an ftp session already in progress.
+First, we list items in a bucket, then we download the contents of the bucket.
+
+.. code-block:: shell
+
+   ftp> ls runner/chunkdocs/metadata
```

Comment:
Shall we include the connection example here too?

Reply:
MIght as well, I suppose. I was trying to keep it shorter.